kernel/nativelib.ml

```diff
@@ -158,7 +158,7 @@ let call_linker ?(fatal=true) prefix f upds =
   (try
     if Dynlink.is_native then Dynlink.loadfile f else !load_obj f;
     register_native_file prefix
-   with Dynlink.Error e as exn ->
+   with Dynlink.Error _e as exn ->
```

Comment:
I would just use `_` here as the name is not informative.

Reply:
Done.